dojo/cred/views.py

```diff
@@ -326,7 +326,7 @@ def view_cred_finding(request, fid, ttid):
         title="Credential Manager", top_level=False, request=request)
     cred_type = "Finding"
     edit_link = None
-    view_link = reverse(
+    reverse(
         'view_cred_finding', args=(
             fid,
             cred.id,
```

Comment:
Agreed with Kiblik ^ here is an example of still doing the unused processing

Reply:
Good catch. I'll go back over these changes a bit more carefully